src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -64,11 +64,13 @@ public static async Task EnsureVirtualEnvrionmentIgnored()
             }
         }
 
-        private static void CreateRequirements()
+        private async static Task CreateRequirements()
         {
             if (!FileSystemHelpers.FileExists(Constants.RequirementsTxt))
             {
-                FileSystemHelpers.WriteAllTextToFile(Constants.RequirementsTxt, Constants.PythonFunctionsLibrary);
+                ColoredConsole.WriteLine($"Writing {Constants.RequirementsTxt}");
+                string requirementsTxtContent = await StaticResources.PythonRequirementsTxt;
+                await FileSystemHelpers.WriteAllTextToFileAsync(Constants.RequirementsTxt, requirementsTxtContent);
             }
```

Comment:
Could you also add functionality in `func host start` for python - which reads the `requirments.txt` and checks for not required dependencies - and throw a warning suggesting the user to remove it?

Reply:
Instead of adding it in `func host start`, I think `func azure functionapp publish` is a better place to warn our users, since requirements.txt is resolved at the time of deployment.